javademo/src/main/java/com/example/javademo/Main.java

```diff
@@ -134,6 +133,37 @@ public static void main(String[] args) throws Exception {
         // Verify a signature
         boolean isCorrect = StarknetCurve.verify(publicKey, hash, r, s);
         System.out.println("Is signature correct? " + isCorrect + ".");
+
+        // Batch RPC requests
+        // Get transactions data
+        HttpBatchRequest<Transaction> transactionsRequest = provider.batchRequests(
+                provider.getTransaction(declareResponse.getTransactionHash()),
+                provider.getTransaction(deployContractResult.transactionHash)
+        );
+
+        List<RequestResult<Transaction>> transactionsResponse = transactionsRequest.send();
+
+        Transaction declareTransaction = transactionsResponse.get(0).getOrNull();
+        Transaction deployContractTransaction = transactionsResponse.get(1).getOrNull();
+
+        System.out.println("Declare transaction: " + declareTransaction);
+        System.out.println("Deploy contract transaction: " + deployContractTransaction);
```

Comment:
nit: Both `declareTransaction` and `deployContractTransaction` produce a significant amount of output. Maybe it would be better to use `getTransactionStatus` or `getTransactionReceipt` if possible, instead of `getTransaction`?

Reply:
Sure, may be a bit better. I changed it to use `getTransactionStatus` 👍